test/common/testinfra.py

```diff
@@ -608,6 +609,27 @@ def sort_key(entry):
     # The details can't contain newline characters and should be brief
 
     def update_known_issue(self, number, err, details, context, timestamp=None):
+        def redact_audit_variables(message):
```

Comment:
Why is this a nested function? It seems like it would be more readable as a separate global function no?